packages/contracts-bedrock/contracts/test/L1Block.t.sol

```diff
@@ -205,20 +240,19 @@ contract L1BlockTest is CommonTest {
         });
     }
 
+    /// @dev Tests that `getL1BlockHash` returns correct values.
     function test_getL1BlockHash_succeeds() external {
```

Comment:
nit: a better name for the test would be `test_getL1BlockHash_overwritten_ring_buffer_succeeds()`

Reply:
cool, I renamed it to test_getL1BlockHash_overwrittenRingBuffer_succeeds because the linter complains